scss/_navbar.scss

```diff
@@ -57,7 +61,107 @@
       @extend %container-flex-properties;
     }
   }
+
+  // Boosted mod
```

Comment:
We may need to add some words to explain this large Boosted mod.

Reply:
Refactored